src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -147,7 +147,7 @@ const ReleaseContent = () => {
                               name,
                             )
                           }
-                          className="govuk-link"
+                          className="govuk-link dfe-align--left"
```

Comment:
Remove the classnames here. It'd be more suitable to make the stytle changed directly in `ButtonText`, as it's supposed to look like a link, but be a button.